test/framework/package.py

```diff
@@ -224,6 +223,30 @@ def test_package(self):
             print "The Package File"
             print read_file(pkgfile)
 
+        toy_txt = read_file(os.path.join(test_easyconfigs, 't', 'toy', 'toy-0.0-gompi-1.3.12-test.eb'))
+        replace_str = '''description = """Toy C program. Now with `backticks' ''' "\n" '''and newlines"""'''
```

Comment:
since you're using triple quotes, you can just insert a hard newline, but then it looks icky due to left indent

you can also:

```python
replace_str = '''description = """Toy C program. Now with `backticks'\n''''
replace_str += '''and newlines"""'''
```